graphium/utils/spaces.py

```diff
@@ -105,7 +108,7 @@
     "msle": TorchMetrics.mean_squared_log_error,
     "pearsonr": TorchMetrics.pearson_corrcoef,
     "spearmanr": TorchMetrics.spearman_corrcoef,
-    "r2": TorchMetrics.r2_score,
+    "r2_score": TorchMetrics.r2_score,
```

Comment:
Please do a ctrl+F to change all `r2` in yaml files to `r2_score`

Reply:
interestingly `r2_score` was already used in many places